paddle/fluid/operators/elementwise/elementwise_min_op.cc

```diff
@@ -53,6 +53,25 @@ class ElementwiseMinOpMaker : public ElementwiseOpMaker {
   }
 };
 
+class ElementwiseFMinOpMaker : public ElementwiseOpMaker {
+ protected:
+  std::string GetName() const override { return "FMin"; }
+  std::string GetEquation() const override { return "Out = fmin(X, Y)"; }
+
+  void AddInputX() override {
+    AddInput("X", "The first tensor holding the elements to be compared.");
+  }
+
+  void AddInputY() override {
+    AddInput("Y", "The second tensor holding the elements to be compared.");
+  }
+
+  std::string GetOpFuntionality() const override {
+    return "Compare two tensors and returns a new tensor containing the "
+           "element-wise minima.";
```

Comment:
what's different between fmin and min, shall we describe it more detailed？

Reply:
OK